cst438assignment1/src/main/java/com/example/cst438assignment1/Hello.java

```diff
@@ -17,28 +16,29 @@
   MovieRepository movieRepository;
 
   @GetMapping("/movies")
-  public String hello(Model model) {
-    Iterable<Movie> movies = movieRepository.findAll();
+  public String movieList(Model model) {
+    Iterable<Movie> movies = movieRepository.findAllMoviesOrderByTitleDateDesc();
     model.addAttribute("movies", movies);
     return "movie_list";
   }
 
   @PostMapping("/movies/new")
-  public String processPersonForm(@Valid Movie movie,
+  public String processMovieForm(@Valid Movie movie,
       BindingResult result,
       Model model) {
     if (result.hasErrors()) {
       return "movie_form";
     }
 
+    movie.setTime(new java.util.Date().toString());
     movieRepository.save(movie);
     Iterable<Movie> movies = movieRepository.findAll();
```

Comment:
This can be created into a method.  The same code is used in lines 20-21.  This is not a big deal.